### PR TITLE
Sync with master branch

### PR DESCRIPTION
**Pull request by JafarAbdi**

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,126 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
```
Comment (JafarAbdi) [inline .github/workflows/ci.yaml:1]: Remove

File: .github/workflows/downstream.rosinstall [hunk 2]
```diff
@@ -0,0 +1,16 @@
+- git:
```
Comment (JafarAbdi) [inline .github/workflows/downstream.rosinstall:1]: Remove

File: .github/workflows/prerelease.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+# This config uses industrial_ci (https://github.com/ros-industrial/industrial_ci.git).
```
Comment (JafarAbdi) [inline .github/workflows/prerelease.yaml:1]: Remove

File: .github/workflows/upstream.rosinstall [hunk 4]
```diff
@@ -0,0 +1,8 @@
+- git:
```
Comment (JafarAbdi) [inline .github/workflows/upstream.rosinstall:1]: Remove

File: README.md [hunk 5]
```diff
@@ -6,10 +6,9 @@ The MoveIt Motion Planning Framework for **ROS 2**. For ROS 1, see [MoveIt 1](ht
 
 ## Continuous Integration Status
 
+
 [![Formatting (pre-commit)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/format.yaml?query=branch%3Amain)
 [![Build and Test (Foxy)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test.yaml?query=branch%3Amain)
-[![Build and Test (Rolling)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test_rolling.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test_rolling.yaml?query=branch%3Amain)
-[![Build and Test (Galactic)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test_galactic.yaml/badge.svg?branch=main)](https://github.com/ros-planning/moveit2/actions/workflows/build_and_test_galactic.yaml?query=branch%3Amain)
```
Comment (JafarAbdi) [inline README.md:12]: Revert

File: moveit_core/CMakeLists.txt [hunk 6]
```diff
@@ -203,7 +237,6 @@ if(BUILD_TESTING)
   set(ament_cmake_cppcheck_FOUND TRUE)
   set(ament_cmake_cpplint_FOUND TRUE)
   set(ament_cmake_uncrustify_FOUND TRUE)
-  set(ament_cmake_lint_cmake_FOUND TRUE)
```
Comment (JafarAbdi) [inline moveit_core/CMakeLists.txt:206]: Revert

File: moveit_core/collision_detection_fcl/src/collision_env_fcl.cpp [hunk 7]
```diff
@@ -50,6 +50,29 @@ namespace collision_detection
 static const rclcpp::Logger LOGGER = rclcpp::get_logger("moveit_collision_detection_fcl.collision_env_fcl");
 const std::string CollisionDetectorAllocatorFCL::NAME("FCL");
 
+namespace
+{
+// Check whether this FCL version supports the requested computations
+void checkFCLCapabilities(const DistanceRequest& req)
+{
+#if MOVEIT_FCL_VERSION < FCL_VERSION_CHECK(0, 6, 0)
+  static rclcpp::Clock clock;
```
Comment (JafarAbdi) [inline moveit_core/collision_detection_fcl/src/collision_env_fcl.cpp:59]: Should be RCL_ROS_TIME

**Comment by henningkayser** [comment]
@JafarAbdi this looks really good so far. Please fix CI by adding the `robot_state_publisher` release to the rolling `.repos` file (it's already been bloomed and just needs to be synced https://github.com/ros/rosdistro/pull/29858)

**Comment by JafarAbdi** [comment]
> @JafarAbdi this looks really good so far. Please fix CI by adding the `robot_state_publisher` release to the rolling `.repos` file (it's already been bloomed and just needs to be synced [ros/rosdistro#29858](https://github.com/ros/rosdistro/pull/29858)). [This tag](https://github.com/ros2-gbp/robot_state_publisher-release/releases/tag/debian%2Fros-rolling-robot-state-publisher_2.5.0-1_focal) should work for this.

This shouldn't fail since It's already synced, see http://packages.ros.org/ros2/ubuntu/pool/main/r/ros-rolling-robot-state-publisher/

**Comment by henningkayser** [comment]
The `main` branch doesn't have an issue with this :thinking: https://github.com/ros-planning/moveit2/runs/2932469721#step:6:942

**Comment by JafarAbdi** [comment]
@vatanaksoytezer Did debug it and it turns out `ros-testing` somehow broke there

**Comment by vatanaksoytezer** [comment]
@henningkayser I think this is a recent issue. I cannot compile the main branch in my fork of moveit2 for rolling-testing (see https://github.com/vatanaksoytezer/moveit2/runs/2934755305?check_suite_focus=true). rolling-main seems fine. I can also reproduce robot-state-publisher not being exist if I change the repos with ros-testing repos on my local rolling setup. I'll go ahead and open a new issue on this.

**Comment by JafarAbdi** [comment]
Look like the failure happened after https://github.com/ros/rosdistro/pull/30045 got merged

**Comment by vatanaksoytezer** [comment]
They just fixed the issue (see https://github.com/ros/robot_state_publisher/issues/167). Re-running the job should be sufficient now.

**Comment by henningkayser** [comment]
> They just fixed the issue (see [ros/robot_state_publisher#167](https://github.com/ros/robot_state_publisher/issues/167)). Re-running the job should be sufficient now.

Looks like that re-run fixed it. I'm just done reviewing the diff and didn't find anything suspicious. After testing this locally, I'll approve and fast-forward merge to main.

**Review by henningkayser — APPROVED** [review]

**Pull request merged**